br/pkg/checkpoint/checkpoint.go

```diff
@@ -137,107 +141,6 @@ type ChecksumInfo struct {
 	DureTime time.Duration `json:"dure-time"`
 }
 
-type ChecksumRunner struct {
```

Comment:
If there are many tables spent more than 60 seconds to do checksum, the checksum runner would upload many checksum data file in a short time. Therefore, change it to a fixed duration to upload, we need to be faced that we spent much time to do checksum, but failed within next 5 seconds and the checksum item cannot be persist into the external storage.